src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null || password.length() < 10
+                || repeatPassword.length() < 10 || !password.equals(repeatPassword)) {
+            throw new PasswordValidationException("Wrong passwords");
+        } else {
```

Comment:
else clause is redundant here, return true after if

Reply:
done